src/components/config_profile/test/profile_test.cc

```diff
@@ -724,6 +724,193 @@ TEST_F(ProfileTest, CheckIntContainerInSecurityData) {
 }
 #endif
 
+TEST_F(ProfileTest, CheckSdlVersion) {
+  // Set new config file
+  profile_.set_config_file_name("smartDeviceLink_test.ini");
```

Comment:
@Ypostolov please extract "smartDeviceLink_test.ini" to some variable, to avoid repeating it in multiple test cases

Reply:
@LuxoftAKutsan 
fixed in https://github.com/smartdevicelink/sdl_core/pull/1995/commits/8b04ba878df1f894f8688d2196f7a6c4b29b9328